.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -489,6 +494,10 @@ echo -e "Compile and install analysis tools"
 
 mkdir -p ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools
 
+# WIP - HASKELL BINARY IS NOT AVAILABLE ARM 64
+if [ ${UTM_ARM} == 0 ]; then
```

Comment:
We should be able to download these binaries, though is this comment indicating that cannot run these on ARM? Or did you not try?

Reply:
The comment below was the error message
"only 64-bit (x86_64) Linux binary is available"

I haven't further investigated yet